src/Microsoft.DotNet.ProjectModel/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    internal static class Constants
+    {
+        public static readonly string DefaultOutputDirectory = "bin";
```

Comment:
I'd put these in OutputDirectories.Default and Configuration.Default, respectively. It's hard to rebuild context from generalized Constants classes.